### DISCUSSION TITLE
I've noticed two "Daily Upgrade" tasks in Task Scheduler, are they added by WingetUI?

### DISCUSSION THREAD

**Discussion by jan-oratowski** [Devolutions/UniGetUI#2203] [Q&A]
TL;DR: Are those two entries added there by WingetUI? I'm trying to find what program I should blame for that.

![image](https://github.com/marticliment/WingetUI/assets/54535207/bd079f19-cf8f-4ac0-915f-7ca254cb67f1)

Full story:

I own a few programs that I paid for and received X months of free upgrades, I'm fine with the features they currently have, so I'm not paying any more for updates.

Unfortunately "something" keeps updating them. At first, I blamed WingetUI, but then I made sure that those packages are set to "never update" in WingetUI.
They still were updated, so after some looking around (mainly checking Winget logs) I've noticed something keeps executing every day. That's how I managed to find those two tasks. I can just disable/delete them, but first I want to know what added it there so I know who I should blame. Also since something has added it there, they probably will be added there again.

One of the auto-upgraded programs is Manic Time, thanks to that it stops working and I have to manually uninstall it, find the installer of the correct version, install it, and then set it up to connect with the server again. Every time I have to do it, it takes 15 minutes. TBH I prefer to uninstall/disable whatever is causing auto-update, or if I can't find the cause, completely remove Winget.

Tried googling (actually binging), but found nothing interesting.

**Comment by marticliment** [comment]
No, WinGetUI does not mess with Task Scheduler (yet)
